lib/timeline/component/item/Item.js

```diff
@@ -169,8 +169,13 @@ Item.prototype._repaintDragCenter = function () {
     });
 
     if (this.dom.box) {
-      this.dom.box.appendChild(dragCenter);
-    } 
+      if (this.dom.dragLeft) {
+        this.dom.box.insertBefore(dragCenter, this.dom.dragLeft);
+      }
+      else {
```

Comment:
It does not break it, I will provide an example on monday.

The dragCenter was only added along with the dragLeft and dragRight elements before on selection. It was not added before selection and the dragging still works without it if itemsAlwaysDraggable is enabled.
What I did, is add the dragLeft and dragRight elements before selection if enabled by new option, thus enabling range dragging without selection. The dragCenter is then added on selection before these 2 elements to keep correct z-index.
When the new option is not enabled, all these elements are added on selection and the current behaviour is kept.
I tried it only with the current release though, it's possible it works differently in the develop branch, I'll check it.